Livecheckables/mysql-client.rb

```diff
@@ -0,0 +1,6 @@
+class MysqlClient
+  livecheck do
+    url "https://github.com/mysql/mysql-server.git"
+    regex(/mysql-v?(\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/^mysql-v?(\d+(?:\.\d+)+)$/i)
```

Adding the `$` delimiter at the end of the regex for Git tags makes sure we don't match any potential prerelease versions in the future, if they ever appear (e.g., `1.2.3-rc1`, `1.2.3-alpha1`, `1.2.3-beta1`, etc.). We also use the leading `^` delimiter, to make sure we don't end up matching any weird tags that could pop up in the future (e.g., `weird-mysql-1.2.3`).